### DISCUSSION TITLE
🧩 FEATURE REQUEST: Short description to custom shortcuts

### DISCUSSION THREAD

**Discussion by money8203** [dailydotdev/daily#1005] [Feature Requests]
Feature Request
Give the option to add a short description to custom shortcuts.

Description
For people using assistive technology the new shortcuts use the site URL as the accessible text (the alt attribute on the image within the anchor).

This is not an ideal solution as if the bookmark contains a load of identification characters as part of a key or they have 2 bookmarks for the same site it means they cannot differentiate easily.

Instead being able to have some user defined short text to describe the bookmark would be better.

Solution
Make the modal wider for adding shortcuts and add a second column "bookmark description (optional)".

This should then be used as the alt text on the image if it is set (if not fall back to the URL as alt text as it is currently).

Additionally this would be great if this is set as as the title attribute on the anchor so you can hover over icons for the same site but different endpoints, in order to differentiate them.

Alternate Solutions
Alternatively allowing people to upload their own icons may be an option to solve the differentiation part for 2 links to the samebase domain, it doesn't fix the accessibility issue though.

**Comment by idoshamun** [comment]
Hi @money8203,

Thanks for the suggestion. We will add it to the backlog.

**Comment by karthiknadar1204** [comment]
I would like to work on this issue,please assign it to me.

**Comment by sshanzel** [comment]
Hi @karthiknadar1204 thank you for your interest in contributing to the app! As I see you've been wanting to help on so many open issues, I think it would be best if you filter the issues tab with `Status: Accepted` or `Status: help-wanted` labels. I'd be glad to help you out!
